markdown_stoplight/events.md

```diff
@@ -50,21 +49,23 @@ You’ll receive something like this:
 availabilityText: "Available"
 available: true
 basePrice: {refQuantity: {…}, refPrice: {…}, formatted: "1 m³ = £0,12", quantity: {…}}
-conditionMicrodata: "NewCondition"
-customAttributes: [{…}]
+conditionMicrodata: "ConditionNew"
+customAttributes: null
 deliveryPeriod: "2-3"
 deliveryPeriodUnit: "DAYS"
 description: null
 energyLabel: null
 energyLabelSourceFile: null
+gtin: null
```

Comment:
This is new for both projects, and it can be a string with a gtin number (isbn, ian, etc.)